docs/interactions/Slash_Commands.md

```diff
@@ -151,6 +151,8 @@ This command will only be available within the guild that you specified.
 > warn
 > Guild commands update **instantly**. We recommend you use guild commands for quick testing, and global commands when they're ready for public use.
 
+Take a look at the [application command option data structure][#DOCS_INTERACTIONS_SLASH_COMMANDS/application-command-object-application-command-option-structure] for extensive info about the types of `options` you can use.
```

Comment:
you need to use it like this: `[abc](abc.com)`
but you are using like this: `[abc][abc.com]`
![image](https://user-images.githubusercontent.com/79790819/123050277-726c9180-d3f8-11eb-92f4-83f06548e844.png)

Reply:
Fixed!